src/test/java/io/rocketpartners/cloud/action/sql/TestSqlQuery.java

```diff
@@ -395,6 +395,16 @@ public void test1() throws Exception
                             "select *, `firstName` AS 'name' from table1 LIMIT 100", // 
                             "select *, `firstName` AS 'name' from table1 LIMIT 100"));//the dynamicsql and preparedsql should match without fields
 
+      tests.add(new RqlTest("emp(startYear)", //
```

Comment:
Nice job knowing to add these tests cases.  I did not explicitly ask for them.  This was the right thing to do.